### PR TITLE
FINERACT-840 - Replace demo.openmf.org/api-docs

### PR DESCRIPTION
**Pull request by xurror**
Update API docs and demo app URL

## Description
Someone pointed out on the mailing list that https://demo.openmf.org/api-docs/apiLive.htm, which we currently link to from https://github.com/apache/fineract#apache-fineract-platform-api, is down; quote: "I've checked a number of times over the past couple weeks and it
seems that's always been the case.")

We should fix that.

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

- [ ] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [ ] API documentation at https://github.com/apache/fineract/blob/develop/api-docs/apiLive.htm has been updated with details of any API changes.

- [ ] Integration tests have been created/updated for verifying the changes made.

- [ ] All Integrations tests are passing with the new commits.

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

File: docs/system-architecture/index.html [hunk 1]
```diff
@@ -233,7 +233,7 @@ <h4>History</h4>
             <ul>
                 <li>2006: Project intiated by <a href="http://www.grameenfoundation.org/" target="_blank">Grameen Foundation</a></li>
                 <li>Late 2011: Grameen Foundation handed over full responsibility to open source community.</li>
-                <li>2012: Mifos X platform started. Previous members of project come together under the name of Community for Open Source Microfinance (COSM / <a href="http://www.openmf.org" target="_blank">OpenMF</a>)</li>
+                <li>2012: Mifos X platform started. Previous members of project come together under the name of Community for Open Source Microfinance (COSM / <a href="http://www.mifos.io" target="_blank">OpenMF</a>)</li>
```
Comment (awasum) [inline docs/system-architecture/index.html]: use: `www.mifos.org` here instead of mifos.io (as it does not have any content)

**Review by awasum — CHANGES_REQUESTED** [review]

File: docs/system-architecture/index.html (hunk 1 above)
Comment (xurror) [reply docs/system-architecture/index.html]: alright

**Review by awasum — APPROVED** [review]

**Review by vorburger — APPROVED** [review]

**Comment by xurror** [comment]
I just noticed I failed to update the links in the API docs. Link to reference app in documentation still redirects us  https://demo.openmf.org instead of https://demo.mifos.io

**Comment by awasum** [comment]
> I just noticed I failed to update the links in the API docs. Link to reference app in documentation still redirects us https://demo.openmf.org instead of https://demo.mifos.io

Have u sent a new PR for this? @xurror

**Pull request merged**